randovania/games/samus_returns/logic_database/Area 2 Dam Exterior.txt

```diff
@@ -261,7 +273,11 @@ Extra - asset_id: collision_camera_007
   * Layers: default
   * Tunnel with Bomb Block to Maintenance Tunnel/Tunnel to Fan Funnel
   > Tunnel to Arachnus Arena
-      Lay Any Bomb
+      All of the following:
+          Lay Any Bomb
+          Any of the following:
+              Disabled Final Boss Arachnus
+              Enabled Final Boss Arachnus and All DNA
```

Comment:
Whats the reasoning for this? I'm not following here.

Reply:
To account for entering the room backwards from the tunnel in DLR

Reply:
Feel like it might be cleaner to have those requirements in the arachnus arena itself but this is good enough for now.